modules/category/category.service.js

```diff
@@ -29,15 +32,15 @@ class CategoryService {
   }
 
   async checkCategoryExist(data) {
-    const category = await Category.find({
+    const category = await Category.countDocuments({
```

Comment:
Please, call variable properly, because it is not an actual entity, it is a count